### PR TITLE
solveLCPmex was not being used by the contact sensors 

### PR DESCRIPTION
**Pull request by RussTedrake**
(they were calling solveLCP directly).
save one output call in FeedbackSystem.
as described in #1106.

### CODE REVIEW

File: systems/plants/TimeSteppingRigidBodyManipulator.m [hunk 1]
```diff
@@ -252,6 +248,10 @@ function checkDirty(obj)
     end
 
     function [obj,z,Mqdn,wqdn,dz,dMqdn,dwqdn] = solveLCP(obj,t,x,u)
+      if (nargout<5 && obj.gurobi_present && obj.manip.only_loops && obj.manip.mex_model_ptr~=0 && ~obj.position_control)
+        [obj,z,Mqdn,wqdn] = solveMexLCP(obj,t,x,u);
```
Comment (psiorx) [inline systems/plants/TimeSteppingRigidBodyManipulator.m]: Looks good to me.

File: systems/plants/TimeSteppingRigidBodyManipulator.m [hunk 2]
```diff
@@ -252,6 +248,10 @@ function checkDirty(obj)
...
+      end
```
Comment (psiorx) [inline systems/plants/TimeSteppingRigidBodyManipulator.m]: Wait.  Isn't this still solving LCP twice?  I think we need a return statement here.

Comment (RussTedrake) [inline systems/plants/TimeSteppingRigidBodyManipulator.m]: Crap.  Good catch.

**Comment by psiorx** [comment]
I have a fix for this.  SolveLCPmex needs to return the possible_limit_indices and possible_contact_indices so the rest of the LCP_cache data structures needed by ContactForceTorqueSensor can be populated.  

I'll push on this tomorrow.

**Pull request merged**